src/Console/Command/FixCommand.php

```diff
@@ -347,6 +347,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if ($config->usingCache()) {
+            $cacheFile = $config->getCacheFile();
+            if (file_exists($cacheFile) && null !== $stdErr) {
```

Comment:
`file_exists` will return true if directory with the same name exists. It could lead to problem since we are going to use it like a file.
Please use `is_file`.